app/client-app/src/app/pages/Wordlist/components/WordCard/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import { Box, Badge, Heading, Button, Icon, Stack } from "@chakra-ui/react";
+import { FaPlusCircle, FaMinusCircle } from "react-icons/fa";
+import { ActiveIndicator } from "components/core";
+
+export const WordCard = ({ type, word, isExcluded }) => {
+  const [isBlocked, setIsBlocked] = useState(isExcluded);
+
+  const handleAction = () => {
+    if (isBlocked) {
+      setIsBlocked(false);
+    } else {
+      setIsBlocked(true);
+    }
+  };
```

Comment:
```suggestion
  const handleAction = () => {
    setIsBlocked(!isBlocked);
  };
```
A more shorthand version.